### DISCUSSION TITLE
Mini Changes for Gold Bomb

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2771] [Ideas]
I was given these ideas while I was bored in my bed. Here I pass my ideas

1. The golden bomb when frightened to see tux, has to jump if there is something to jump and get there. Something similar to Haywire.

2. The moment the golden bomb gets scared and runs, when it's on a slope, you might see a 5% chance that the golden bomb will slip and start rolling down the slope, and when it hits a wall, it will explode with its coins that it had inside. Something akin to Igel and Corrupt Igel

I don't know if they're going to be added to the enemy. But if you're interested in any developer or the community, let me know what you think

**Comment by Frostwithasideofsalt** [comment]
Idea one isn't that bad of an idea but randomness is horrible and should be used as little as possible

**Comment by tylerandari13** [comment]
The second one feels straight out of Pizza Tower lmao.

**Reply by VincentAraneo** [reply]
LOL
